src/skin/css/firstRun.css

```diff
@@ -788,6 +788,12 @@ body {
   background-image:linear-gradient(#DFDFE6, #DFDFE6 30%, #fff 30%);
   padding:4rem 0;
 }
+#pb-privacy-policy {
+  color:#F06A0A;
+  font-family:"OpenSans",sans-serif;
```

Comment:
We ship Privacy Badger with OpenSans light and bold fonts only ... Not sure if this works correctly.